modules_extra/network.js

```diff
@@ -100,77 +95,166 @@ function duration (s) {
     return round(s)+'ms'
 }
 
+function peerListSort (a, b) {
```

Comment:
correct. I was wondering about using a filter instead of sort. I'm still pretty interested in making different sections to make this all easier to read / parse. 

What do you reckon @mmckegg